diff --git a/node_modules/@stream-io/flat-list-mvcp/android/build.gradle b/node_modules/@stream-io/flat-list-mvcp/android/build.gradle
index 0cd8851..0028db4 100644
--- a/node_modules/@stream-io/flat-list-mvcp/android/build.gradle
+++ b/node_modules/@stream-io/flat-list-mvcp/android/build.gradle
@@ -4,7 +4,7 @@ buildscript {
 
   repositories {
     google()
-    jcenter()
+    mavenCentral()
   }
 
   dependencies {
@@ -52,7 +52,6 @@ android {
 
 repositories {
   mavenCentral()
-  jcenter()
   google()
 
   def found = false
